curiouslearning/FeedTheMonsterJS
# Dev optimization to Develop Code merge

# Changes
- Fixed incorrect object value used for displaying game level text at each game level button.

# How to test
- Run FTM and in the start scene press the dev button to enable developer mode.
- Press the start button and in the level selection, all levels should be unlocked with the text level displayed at each level buttons.

Ref: [JIRA-CODE](https://curiouslearning.atlassian.net/browse/FM-256)
